### PR TITLE
Tools cleanup

### PR DESCRIPTION
**Pull request by davidhesselbom**
Cleaned up tools scripts, leaving fewer distracting files in the main folder.

### CODE REVIEW

**Comment by davidhesselbom** [comment]
Looks like I made some incorrect assumptions "cleaning up" `install_rock.sh`. Let me try again...

**Comment by davidhesselbom** [comment]
@jonathanudd Peer review?

**Comment by jonathanudd** [comment]
We are not running appveyor any more right? If we don't do we need to keep the windows part in for installing rock?

**Comment by davidhesselbom** [comment]
I guess not. I can remove it.

**Comment by jonathanudd** [comment]
Other than that I think it looks good.

File: install_rock.sh [hunk 1]
```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-if [ "$(uname -o)" == "Msys" ]; then
-	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*zip\>')"
-	wget https://github.com/cogneco/rock/releases/$tmp
-	echo ${tmp##*/}
-	7z e ${tmp##*/}
-else
```
Comment (marcusnaslund) [inline install_rock.sh]: I don't know about this. @simonmika - do we want to get rid of everything related to Windows?

Comment (davidhesselbom) [inline install_rock.sh]: As far as the script goes, it's all versioned, so we can revive if need be (although if the file is moved or renamed again, it'll be harder to find this part of it).

Comment (davidhesselbom) [inline install_rock.sh]: If we're dropping Windows support though, should we still keep support for running without a GPU?

Comment (simonmika) [inline install_rock.sh]: We should keep support for Windows as long as the impact on our speed forward is low. We might need it soon again.

Support for running without GPU is probably even more interesting, long term.

Comment (davidhesselbom) [inline install_rock.sh]: Okay, I'll back up my branch and remove the last commit from the PR.

Comment (marcusnaslund) [inline install_rock.sh]: Good good.

Comment (davidhesselbom) [inline install_rock.sh]: And so I did, a long time ago :)

Comment (marcusnaslund) [inline install_rock.sh]: > And so I did, a long time ago :)

Oh, right, so you did. I was expecting some news below this, but I realize that was incorrect of me.

**Comment by marcusnaslund** [comment]
What else is supposed to be in `tools`, btw? Does this need a separate `ci` folder?

**Comment by davidhesselbom** [comment]
Not sure, I just wanted some consistency across projects. I can move it elsewhere, if you want.

**Comment by davidhesselbom** [comment]
Memtest could go in `tools`, I suppose.

**Comment by marcusnaslund** [comment]
Right, it's the same in vidproc. OK, that's fine by me.

**Comment by davidhesselbom** [comment]
What is this "vidproc" you speak of?
